RadarSDK/Radar.m

```diff
@@ -446,15 +456,19 @@ + (void)sendLogConversionRequestWithName:(NSString * _Nonnull) name
 }
 
 + (void)logOpenedAppConversion {
-    // if opened_app has been logged within the last second, don't log it again
-    NSTimeInterval lastAppOpenTimeInterval = [[NSDate date] timeIntervalSinceDate:[RadarSettings lastAppOpenTime]];
-    if (lastAppOpenTimeInterval > 1) {
-        [RadarSettings updateLastAppOpenTime];
-        [self sendLogConversionRequestWithName:@"opened_app" metadata:nil completionHandler:^(RadarStatus status, RadarEvent * _Nullable event) {
-            NSString *message = [NSString stringWithFormat:@"Conversion name = %@: status = %@; event = %@", event.conversionName, [Radar stringForStatus:status], event];
-            [[RadarLogger sharedInstance] logWithLevel:RadarLogLevelInfo message:message];
-        }];
-    }
+    // Perform a non-blocking sleep for 1 second before starting
+    dispatch_after(dispatch_time(DISPATCH_TIME_NOW, (int64_t)(1 * NSEC_PER_SEC)), dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_DEFAULT, 0), ^{
+        // if opened_app has been logged within the last second, don't log it again
```

Comment:
```suggestion
        // if opened_app has been logged within the last  two seconds, don't log it again
```

^that's what it needs to be per the logic on L464, right.

Reply:
Yes. Honestly not the most "academic" solution to this issue, but favoring it right now for its simplicity.